webrtc-stats.html

```diff
@@ -1264,14 +1264,14 @@
                 </p>
               </dd>
               <dt>
-                <dfn><code>averageRtt</code></dfn> of type <span class=
+                <dfn><code>totalRtt</code></dfn> of type <span class=
                 "idlMemberType"><a>double</a></span>
               </dt>
               <dd>
                 <p>
-                  Represents the average RTT measured in seconds since the beginning of the session,
-                  computed from both STUN connectivity checks [[!STUN-PATH-CHAR]] and consent
-                  responses [[!RFC7675]].
+                  Represents the total RTT measured in seconds since the beginning of the session,
```

Comment:
May be valuable to elaborate a bit more, for clarity. We could replace "the total RTT" with "the sum of all RTT measurements", and explain how this can be used to derive the average, since that will probably be the most common use.